regulations/static/regulations/js/source/views/comment-review-view.js

```diff
@@ -56,6 +60,16 @@ var CommentReviewView = Backbone.View.extend({
     });
     if (target) {
       target.remove();
+
+      this.numberOfComments--;
+      this.checkForComments();
+    }
+  },
+
+  checkForComments: function() {
+    if (this.numberOfComments === 0) {
```

Comment:
Is there a way to do this via inspection instead of having a separate variable? I think it'd be very easy to get out of sync. Would it make sense to remove the entry from `this.comments` inside `clearComment` and then use `this.comments.length`?